Frontend/ui-customer/src/Screens/Types.purs

```diff
@@ -992,3 +994,11 @@ derive instance genericFlowStatusData :: Generic FlowStatusData _
 instance showFlowStatusData :: Show FlowStatusData where show = genericShow
 instance encodeFlowStatusData :: Encode FlowStatusData where encode = defaultEncode
 instance decodeFlowStatusData :: Decode FlowStatusData where decode = defaultDecode
+
+data ZoneType = METRO | HOSPITAL | AIRPORT | SCHOOL | RAILWAY | NOZONE
```

Comment:
@ashishsingh101  How are you maintaining Drop And Pickup in just METRO type
Shouldn't it be METRO_DROP and METRO_PICKUP